ajax/libs/antimoderate/package.json

```diff
@@ -16,15 +16,13 @@
       ]
     }
   ],
-  "license": {
-    "type": "See README.md",
-    "url": "https://github.com/whackashoe/antimoderate#license"
-  },
+  "license": "MIT",
```

Comment:
From the readme, looks like `StackBlur Algorithm` is under MIT but its self may be CC-0 or public domain?

Reply:
Though it is MIT in bower.json, but it'll be great that if you can open an issue ticket to confirm with its author about the license, would you?

Reply:
Sure, I made an issue in the repo there to confirm the license -> https://github.com/whackashoe/antimoderate/issues/15